Resources/Prototypes/_White/Entities/Clothing/OuterClothing/modsuits.yml

```diff
@@ -0,0 +1,48 @@
+﻿- type: entity
+  parent: ClothingOuterBase
+  id: ClothingModsuitChestplateInfiltrator
+  name: infiltrator modsuit chestplate
+  description: A special modular suit cover designed for compact folding inside modular suit control.
```

Comment:
"A purpose-built suit cover that can be neatly folded and store in the infiltrator suit's main unit."